srcv2/components/AdvancedWalletSetup.js

```diff
@@ -116,6 +114,7 @@ const AdvancedWalletSetup = () => {
                         type='text'
                         placeholder={r.label}
                         value={r.value}
+                        onChange={(e) => { r.setValue(e.target.value) }}
```

Comment:
@Ovifer13 why this change?

Reply:
When I did this I forgot to add the onChange value to the input, the input wasn't doing anything till I added this. That's why Oscar was getting that error the other day in the meeting that the button wasn't getting enabled.